controller/test/sensor_tests/SensorTests.cpp

```diff
@@ -164,42 +164,42 @@ TEST(SensorTests, FullScaleReading) {
 // you update the expected values accordingly.
 TEST(SensorTests, TestPositiveVolumetricFlowCalculation) {
   sensors_init();
-  float volumFlow = pressure_delta_to_volumetric_flow(1.0f);
+  float volumFlow = pressure_delta_to_flow(1.0f);
   // 1 kPa differential pressure should result in 9.52e-4 [m^3/s] of Q
   EXPECT_NEAR(volumFlow, 9.52e-4f, COMPARISON_TOLERANCE_FLOW);
 }
 
 TEST(SensorTests, TestNegativeVolumetricFlowCalculation) {
   sensors_init();
-  float volumFlow = pressure_delta_to_volumetric_flow(-1.0f);
+  float volumFlow = pressure_delta_to_flow(-1.0f);
   // -1 kPa differential pressure should result in -9.52e-4 [m^3/s] of Q
   EXPECT_NEAR(volumFlow, -9.52e-4f, COMPARISON_TOLERANCE_FLOW);
 }
 
 TEST(SensorTests, TestZeroVolumetricFlowCalculation) {
   sensors_init();
-  float volumFlow = pressure_delta_to_volumetric_flow(0.0f);
+  float volumFlow = pressure_delta_to_flow(0.0f);
   // 0 kPa differential pressure should result in 0 [m^3/s] of Q
   EXPECT_NEAR(volumFlow, 0.0f, COMPARISON_TOLERANCE_FLOW);
 }
 
 TEST(SensorTests, TestNearZeroVolumetricFlowCalculation) {
   sensors_init();
-  float volumFlow = pressure_delta_to_volumetric_flow(1.0e-7f);
+  float volumFlow = pressure_delta_to_flow(1.0e-7f);
   // 1e-7 kPa differential pressure should result in 3.07e-7 [m^3/s] of Q
   EXPECT_NEAR(volumFlow, 3.07e-7f, COMPARISON_TOLERANCE_FLOW);
 }
 
 TEST(SensorTests, TestLargeVolumetricFlowCalculation) {
   sensors_init();
-  float volumFlow = pressure_delta_to_volumetric_flow(100.0f);
+  float volumFlow = pressure_delta_to_flow(100.0f);
```

Comment:
Nit: I put volumetric in implicitly so as to be abundantly clear that this is volumetric flow, not flow velocity, which is different.

Reply:
Oh, you wouldn't call the other one "velocity"?

Reply:
They both are a velocity, but there are subtle differences, they are not equivalent. Flow velocity is used to calculate the volumetric flow.

Reply:
Are you telling me my physics teacher LIED to me when she said that velocity is defined as dPosition/dT (and therefore has units m/s)?  Like, volumetric flow has units m^3/s, which is different!  (Whereas flow velocity is indeed a vector field with units m/s.)

Destroying my childhood.  :(